charts/partials/templates/_proxy.tpl

```diff
@@ -119,8 +125,14 @@ lifecycle:
         - -c
         - sleep {{.Values.global.proxy.waitBeforeExitSeconds}}
 {{- end }}
-{{- if or (not .Values.global.proxy.disableIdentity) (.Values.global.proxy.saMountPath) }}
+{{- if or (not .Values.global.proxy.disableIdentity) (.Values.global.proxy.saMountPath) (.Values.global.proxy.trace) }}
 volumeMounts:
+{{- if (.Values.global.proxy.trace) }}
+{{- if (.Values.global.proxy.trace.collectorSvcAddr)}}
+- name: podinfo
+  mountPath: var/run/linkerd/podinfo
```

Comment:
Please swap these lines to follow the same alphabetical ordering the other mounts have :wink: